BuffTrigger.lua

```diff
@@ -296,70 +295,84 @@ do
 end
 WeakAuras.aura_cache = aura_cache;
 
-function WeakAuras.SetAuraVisibility(id, triggernum, data, active, unit, duration, expirationTime, name, icon, count, cloneId, index, spellId)
-  local region;
-  local showClones;
+function WeakAuras.SetAuraVisibility(id, triggernum, cloneId, inverse, active, unit, duration, expirationTime, name, icon, count, index, spellId)
+  local triggerState = WeakAuras.GetTriggerStateForTrigger(id, triggernum);
 
-  if(cloneId) then
-    if(data.numAdditionalTriggers > 0) then
-      showClones = data.region.toShow and true or false;
-    else
-      showClones = true;
-    end
-    region = WeakAuras.EnsureClone(id, cloneId);
-  else
-    region = data.region;
```

Comment:
The old code for showClones was rather strange. If you had a aura with: 1.trigger being a multi trigger, 2. trigger being a always inactive trigger and set the combine mode to "OR", then the multi trigger would never show anything. This now works just like I would expect it.